database/entity/0060-update_communities_table/Community.ts

```diff
@@ -25,13 +25,13 @@ export class Community extends BaseEntity {
   endPoint: string
 
   @Column({ name: 'last_announced_at', type: 'datetime', nullable: true })
-  lastAnnouncedAt: Date
+  lastAnnouncedAt: Date | null
 
   @Column({ name: 'verified_at', type: 'datetime', nullable: true })
-  verifiedAt: Date
+  verifiedAt: Date | null
```

Comment:
Not certain if ``| null`` could be replaced with 
```suggestion
  verifiedAt?: Date
```

Reply:
see above